File: uikit/src/main/java/com/sendbird/uikit/activities/viewholder/EmojiViewHolder.java
package com.sendbird.uikit.activities.viewholder;

import androidx.annotation.NonNull;

import com.sendbird.android.Emoji;
import com.sendbird.uikit.databinding.SbViewEmojiBinding;

public class EmojiViewHolder extends BaseViewHolder<Emoji> {
    @NonNull
    private final SbViewEmojiBinding binding;

    public EmojiViewHolder(@NonNull SbViewEmojiBinding binding) {
        super(binding.getRoot());
        this.binding = binding;
    }

    @Override
    public void bind(@NonNull Emoji item) {
        binding.emojiView.drawEmoji(item);
    }
}

File: uikit/src/main/java/com/sendbird/uikit/internal/ui/viewholders/EmojiViewHolder.kt
package com.sendbird.uikit.internal.ui.viewholders

import com.sendbird.android.message.Emoji
import com.sendbird.uikit.activities.viewholder.BaseViewHolder
import com.sendbird.uikit.databinding.SbViewEmojiBinding

internal class EmojiViewHolder(
    private val binding: SbViewEmojiBinding
) : BaseViewHolder<Emoji>(binding.root) {
    override fun bind(item: Emoji) {
        binding.emojiView.drawEmoji(item)
    }
}
